crates/shared/src/arguments.rs

```diff
@@ -270,6 +274,9 @@ impl Display for Arguments {
             self.disabled_one_inch_protocols
         )?;
         writeln!(f, "one_inch_url: {}", self.one_inch_url)?;
+        write!(f, "one_inch_referrer_address: ")?;
+        display_option(&self.one_inch_referrer_address.map(|a| format!("{a:?}")), f)?;
```

Comment:
This was needed because the `Display` implementation of `H160` prints something like `0x123...abc` which is not very helpful.

Reply:
Fun fact, you can also use `{:#x}`.